diff --git a/node_modules/react-native/ReactAndroid/src/main/java/com/facebook/react/modules/statusbar/StatusBarModule.kt b/node_modules/react-native/ReactAndroid/src/main/java/com/facebook/react/modules/statusbar/StatusBarModule.kt
index dbae14c..ba882a8 100644
--- a/node_modules/react-native/ReactAndroid/src/main/java/com/facebook/react/modules/statusbar/StatusBarModule.kt
+++ b/node_modules/react-native/ReactAndroid/src/main/java/com/facebook/react/modules/statusbar/StatusBarModule.kt
@@ -7,12 +7,9 @@
 
 package com.facebook.react.modules.statusbar
 
-import android.animation.ArgbEvaluator
-import android.animation.ValueAnimator
 import android.os.Build
 import android.view.View
 import android.view.WindowInsetsController
-import android.view.WindowManager
 import com.facebook.common.logging.FLog
 import com.facebook.fbreact.specs.NativeStatusBarManagerAndroidSpec
 import com.facebook.react.bridge.GuardedRunnable
@@ -35,19 +32,13 @@ internal class StatusBarModule(reactContext: ReactApplicationContext?) :
   @Suppress("DEPRECATION")
   override fun getTypedExportedConstants(): Map<String, Any> {
     val currentActivity = reactApplicationContext.currentActivity
-    val statusBarColor =
-        currentActivity?.window?.statusBarColor?.let { color ->
-          String.format("#%06X", 0xFFFFFF and color)
-        } ?: "black"
     return mapOf(
         HEIGHT_KEY to PixelUtil.toDIPFromPixel(getStatusBarHeightPx(currentActivity).toFloat()),
-        DEFAULT_BACKGROUND_COLOR_KEY to statusBarColor,
+        DEFAULT_BACKGROUND_COLOR_KEY to "black",
     )
   }
 
-  @Suppress("DEPRECATION")
   override fun setColor(colorDouble: Double, animated: Boolean) {
-    val color = colorDouble.toInt()
     val activity = reactApplicationContext.getCurrentActivity()
     if (activity == null) {
       FLog.w(
@@ -61,24 +52,9 @@ internal class StatusBarModule(reactContext: ReactApplicationContext?) :
           "StatusBarModule: Ignored status bar change, current activity is edge-to-edge.")
       return
     }
-    UiThreadUtil.runOnUiThread(
-        object : GuardedRunnable(reactApplicationContext) {
-          override fun runGuarded() {
-            val window = activity.window ?: return
-            window.addFlags(WindowManager.LayoutParams.FLAG_DRAWS_SYSTEM_BAR_BACKGROUNDS)
-            if (animated) {
-              val curColor = window.statusBarColor
-              val colorAnimation = ValueAnimator.ofObject(ArgbEvaluator(), curColor, color)
-              colorAnimation.addUpdateListener { animator ->
-                activity.window?.statusBarColor = (animator.animatedValue as Int)
-              }
-              colorAnimation.setDuration(300).startDelay = 0
-              colorAnimation.start()
-            } else {
-              window.statusBarColor = color
-            }
-          }
-        })
+    FLog.w(
+        ReactConstants.TAG,
+        "StatusBarModule: Ignored status bar color change because Android edge-to-edge should not set system bar colors directly.")
   }
 
   override fun setTranslucent(translucent: Boolean) {
diff --git a/node_modules/react-native/ReactAndroid/src/main/java/com/facebook/react/views/view/WindowUtil.kt b/node_modules/react-native/ReactAndroid/src/main/java/com/facebook/react/views/view/WindowUtil.kt
index 60f2a94..422065a 100644
--- a/node_modules/react-native/ReactAndroid/src/main/java/com/facebook/react/views/view/WindowUtil.kt
+++ b/node_modules/react-native/ReactAndroid/src/main/java/com/facebook/react/views/view/WindowUtil.kt
@@ -7,7 +7,6 @@
 
 package com.facebook.react.views.view
 
-import android.graphics.Color
 import android.os.Build
 import android.view.Window
 import android.view.WindowManager
@@ -17,15 +16,6 @@ import androidx.core.view.WindowInsetsCompat
 import androidx.core.view.WindowInsetsControllerCompat
 import com.facebook.react.views.common.UiModeUtils
 
-// The light scrim color used in the platform API 29+
-// https://cs.android.com/android/platform/superproject/+/master:frameworks/base/core/java/com/android/internal/policy/DecorView.java;drc=6ef0f022c333385dba2c294e35b8de544455bf19;l=142
-internal val LightNavigationBarColor = Color.argb(0xe6, 0xFF, 0xFF, 0xFF)
-
-// The dark scrim color used in the platform.
-// https://cs.android.com/android/platform/superproject/+/master:frameworks/base/core/res/res/color/system_bar_background_semi_transparent.xml
-// https://cs.android.com/android/platform/superproject/+/master:frameworks/base/core/res/remote_color_resources_res/values/colors.xml;l=67
-internal val DarkNavigationBarColor = Color.argb(0x80, 0x1b, 0x1b, 0x1b)
-
 /**
  * This does not enable or apply edge-to-edge behavior, it simply tracks whether it has been flagged
  * as enabled elsewhere in the application.
@@ -112,14 +102,6 @@ internal fun Window.enableEdgeToEdge() {
     isNavigationBarContrastEnforced = true
   }
 
-  statusBarColor = Color.TRANSPARENT
-  navigationBarColor =
-      when {
-        Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q -> Color.TRANSPARENT
-        Build.VERSION.SDK_INT >= Build.VERSION_CODES.O && !isDarkMode -> LightNavigationBarColor
-        else -> DarkNavigationBarColor
-      }
-
   WindowInsetsControllerCompat(this, decorView).run {
     isAppearanceLightNavigationBars = !isDarkMode
   }
